.github/workflows/quality-gates.yml

```
@@ -0,0 +1,490 @@
+  pull_request:
+    branches: [main, develop]
+  workflow_run:
+    workflows: ["Comprehensive Test Suite", "Coverage Analysis", "Performance Monitoring", "Security Scanning"]
+    types: [completed]
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+    inputs:
```

actionlint (1.7.8)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)